app/components/degree_qualification_cards_component.html.erb

```diff
@@ -29,6 +29,15 @@
                 <dt class="app-qualification__key">Comparability</dt>
                 <dd class="app-qualification__value"><%= naric_statement(degree) %></dd>
               <% end %>
+
+              <% if show_hesa_codes? %>
+                <dt class="app-qualification__key">HESA codes</dt>
+                <dd class="app-qualification__value">
```

Comment:
Can we have this output the following HTML:

```
<dt class="app-qualification__key">HESA codes</dt>
<dd class="app-qualification__value app-qualification__value--grey govuk-!-margin-bottom-0">Type: 72</dd>
<dd class="app-qualification__value app-qualification__value--grey govuk-!-margin-bottom-0">Subject: 100121</dd>
<dd class="app-qualification__value app-qualification__value--grey govuk-!-margin-bottom-0">Establishment: 0012</dd>
<dd class="app-qualification__value app-qualification__value--grey govuk-!-margin-bottom-0">Class: 01</dd>
```

Reply:
Thanks! Tried with multiple `dd`s first - should have realised I only needed to override the margin to get it looking right.